newIDE/app/src/ProjectsStorage/LocalFileStorageProvider/LocalProjectWriter.js

```diff
@@ -142,6 +168,12 @@ export const onSaveProject = (
   };
 
   const projectPath = path.dirname(filePath);
+
+  deleteExistingFilesFromDirs(projectPath).catch(err => {
+    console.error('Unable to delete files in the project:', err);
```

Comment:
I think here we could gain clarity if you mention the context: `Unable to clean project folder before saving project:`